### DISCUSSION TITLE
Connect to remote vscode instance running on host

### DISCUSSION THREAD

**Discussion by niemiaszek** [microsoft/vscode-jupyter#13352] [Questions and Answers]
I work in hybrid mode and I have such setup:
1. Beefy server
2. Linux workstation in the office
3. Windows Laptop which I use on remote work

Using Jupyter Notebook / Jupyter Lab server I would normally run the Jupyter server on host (1) in some kind of persistent terminal (like TMUX) and then I would use my workstation (2) or Laptop (3) to connect to them. 

However, I'm wondering what to do in scenario, when I ran jupyter in remote ssh from workstation to server (2 to 1) and I want to access this instance running on my laptop (3). Obviously, when I connect with remote ssh from laptop to server, I don't see status of currently ran notebook instance from workstation.

In other words, I have few days long process running in vscode jupyter started with remote ssh from 2 to 1 and I want to use the same instance from my laptop 3. 

When I check process information about this VS code instance through htop on 1:
`venv/bin/python -m ipykernel_launcher --ip=127.0.0.1 --stdin=9028 --control=9026 --hb=9025 --Session.signature_scheme="hmac-sha256" --Session.key=b"****" --shell=9027 --transport="tcp" --iopub=9029 --f=****/.local/share/jupyter/runtime/kernel-v2-****.json`

I found some similar questions, but somehow these questions weren't related to my issue (i.e. most people used regular jupyter server on host, not vscode jupyter).
